### PR TITLE
Frontend fixes

### PR DESCRIPTION
**Pull request by robinpiets**
- Fix: updated errors on Location Page
- Feat: Added support for old URL + changed copy

### CODE REVIEW

File: apps/client/src/config.js [hunk 1]
```diff
@@ -100,6 +100,12 @@ const topics = [
   },
 ];
 
+// Support for the old URL ("zonnepanelen-of-warmtecollectoren-plaatsen")
+topics.push({
+  ...topics.find((t) => t.slug === "zonnepanelen-of-zonneboiler-plaatsen"),
+  slug: "zonnepanelen-of-warmtecollectoren-plaatsen",
+});
+
```
Comment (afjlambert) [inline apps/client/src/config.js]: Can we instead do a redirect from old slugs to new ones?

**Review by afjlambert — CHANGES_REQUESTED** [review]

File: apps/client/src/config.js (hunk 1 above)
Comment (robinpiets) [reply apps/client/src/config.js]: See my Slack message in #frontend

Comment (afjlambert) [reply apps/client/src/config.js]: Maybe something like this?

```
diff --git a/apps/client/src/components/Router.js b/apps/client/src/components/Router.js
index 0d82d14..9025bbf 100644
--- a/apps/client/src/components/Router.js
+++ b/apps/client/src/components/Router.js
@@ -1,7 +1,7 @@
 import React from "react";
-import { BrowserRouter, Switch, Route } from "react-router-dom";
+import { BrowserRouter, Switch, Route, Redirect } from "react-router-dom";
 
-import { routeConfig } from "../routes";
+import { routeConfig, redirectConfig } from "../routes";
 import ScrollToTop from "./ScrollToTop";
 
 const Router = (props) => {
@@ -9,6 +9,9 @@ const Router = (props) => {
     <BrowserRouter>
       <ScrollToTop />
       <Switch>
+        {redirectConfig.map(([from, to]) => (
+          <Redirect {...{ from, to }} />
+        ))}
         {routeConfig
           .filter((route) => route.component)
           .map((route, i) => (
diff --git a/apps/client/src/routes.js b/apps/client/src/routes.js
index 0546501..e9c7e26 100644
--- a/apps/client/src/routes.js
+++ b/apps/client/src/routes.js
@@ -27,6 +27,11 @@ export const geturl = (route, params) => {
   return reverse(route, params);
 };
 
+export const redirectConfig = [
+  ["/a", "/b"],
+  ["/c", "/d"],
+];
+
 export const routeConfig = [
   {
     name: "home",
```

Comment (robinpiets) [reply apps/client/src/config.js]: Implemented, thanks mate!

**Review by svenjens — APPROVED** [review]

**Review by afjlambert — APPROVED** [review]

**Pull request merged**